docs/advanced_topics/images/changing_rich_text_representation.rst

```diff
@@ -0,0 +1,57 @@
+.. _changing_rich_text_representation:
+
+=================================
+Changing rich text representation
+=================================
+
+The HTML representation of an image in rich text can be customised - for example, to display captions or custom fields.
+
+To do this requires subclassing ``Format`` (see :ref:`rich_text_image_formats`), and overriding its ``image_to_html`` method.
+
+You may then register formats of your subclass using ``register_image_format`` as usual.
+
+.. code-block:: python
+
+    #image_formats.py
+    from wagtail.images.formats import Format, register_image_format
+
+
+    class SubclassedImageFormat(Format):
+
+        def image_to_html(self, image, alt_text, extra_attributes=None):
+
+            custom_html = #the custom HTML representation of your image here
```

Comment:
Comments "should start with a # and a single space"

https://www.python.org/dev/peps/pep-0008/#inline-comments

Reply:
Oops, sorry. Added now.